plugin/plugins/kafka/service/service_test.go

```diff
@@ -0,0 +1,560 @@
+package service
```

Comment:
The tests in this file look like they require a running kafka broker/zookeeper. If that's the case, they shouldn't be run by default (e.g. with `make test`). But they should still be runnable, possibly with a different make target. Furthermore, they should still be run as part of the github actions and checks, but will probably need a specific job to do so. Also, there should probably be a comment at the top with guidance on running these tests.